util/missingssl.txt

```diff
@@ -19,6 +19,7 @@ SSL_SRP_CTX_free
 SSL_SRP_CTX_init
 SSL_add_dir_cert_subjects_to_stack
 SSL_add_file_cert_subjects_to_stack
+SSL_add_store_cert_subjects_to_stack
```

Comment:
Again no! We should never be adding anything to the "missing" files.

Reply:
Same answer as for the 'X509_STORE_add_` docs